components/Relationships/RelationshipElement.tsx

```diff
@@ -2,35 +2,118 @@ import Link from 'next/link';
 import { RelationshipPerson } from 'types';
 
 interface Props {
-  title: string;
-  data: RelationshipPerson[];
+  type: string;
```

Comment:
Can `type` be literally any string, or is there a valid subset of `types` that could form some kind of union type?

Reply:
There is a valid subset of available types indeed!

I will add the type "RelationshipType" so this is more robust